### ISSUE TITLE
LibreOOPAlgorithm

### ISSUE THREAD

**Issue by jo1st** [tzachi-dar/LibreOOPAlgorithm#17]
I don't know if you remember me but we had a discussion on DXM trip issues and you give me a link or to your plug-in. I cannot find it anymore not really without a PC and I was wondering could you give me a link to it to download it as soon as possible. But the one I already installed in my phone it doesn't seem to connect anymore I wanted to try to reinstall it again

**Comment by tzachi-dar** [comment]
https://drive.google.com/file/d/18zkgZq3ztNlgQZovKcE7ddO70PuawfBB/view

**Comment by tzachi-dar** [comment]
What is the xdrip status? Does it say connected?

**Comment by jo1st** [comment]
Well initially yes but then eventually as soon as I select your plug-in it
becomes disconnected and I don't receive any data anymore

**Comment by tzachi-dar** [comment]
And what happens when you disconnect the plugin? 
Does it become connected again?

**Comment by jo1st** [comment]
exactly. note 9 latest Android, latest xdrip stable. Never had this problem
I don't know if you remember our conversation you said yesterday I used it
with one of the sensor was not behaving correctly. Then I didn't use it for
a while because the same so so we're becoming correctly no problem with
xdrip. And now today new sensor again lot of problems so I decided to use it

**Comment by tzachi-dar** [comment]
When you say that you are not connected. Does it mean that there on xdrip->bluetooth status things get unconnected?

**Comment by jo1st** [comment]
no, the levels number is crossed out, disconnected

**Comment by jo1st** [comment]
Why don't you send me a link to the APK? I want to reinstall it

**Comment by tzachi-dar** [comment]
Sending again (It is already higher) https://drive.google.com/file/d/18zkgZq3ztNlgQZovKcE7ddO70PuawfBB/view

**Comment by jo1st** [comment]
Thank you very much. What's the best way to reinstall this?

**Comment by tzachi-dar** [comment]
Simply remove it and install again.

Can you try and see if libre link works on your phone?

**Comment by jo1st** [comment]
what do you mean by that? which link?

**Comment by tzachi-dar** [comment]
The official abbott application is called librelink you can find it in  the play store. 
Install and see if it works (And what version is it).

**Comment by jo1st** [comment]
just to let you know since today your plug-in keeps popping up warning
asking me to use it to calibrate and no matter how many time I dismiss it I
even disable Force stop, disabled allow notifications still pops up
